corehq/apps/registration/templates/registration/email/confirm_account_reminder.txt

```diff
@@ -0,0 +1,23 @@
+Dear {{ full_name }},
```

Comment:
Yeah, good point. I imagine when they sign up we could store whether they're looking at prelogin in English or French...which would be kind of nice, actually...but doubt we're doing that now.